scripts/ASTImportTest.sh

```diff
@@ -1,10 +1,41 @@
 #!/usr/bin/env bash
+# ------------------------------------------------------------------------------
+# vim:ts=4:et
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) solidity contributors.
+# ------------------------------------------------------------------------------
+
+set -euo pipefail
+
+REDIRECT_TO="/dev/null"
+IMPORT_TEST_TYPE=
+for PARAM in "$@"
+do
+  case "${PARAM}" in
+    ast) IMPORT_TEST_TYPE="ast" ;;
+    --show-errors) REDIRECT_TO="/dev/stderr"
```

Comment:
I believe each case clause should be terminated in `;;`, `;&` or `;;&`, no?
```suggestion
    --show-errors) REDIRECT_TO="/dev/stderr" ;;
```

Reply:
true!